api/models/Playlist.js

```diff
@@ -0,0 +1,25 @@
+/**
+* Playlist.js
+*/
+
+module.exports = {
+
+  attributes: {
+    name: {
+      type: 'string',
+      size: 50
+    },
+    permissions: {
+      type: 'string',
+      defaultsTo: 'public'
+    },
+
+    owner: {
+      model: 'user'
+    },
+    songs: {
+      collection: 'song',
+      via: 'playlists'
```

Comment:
This is how I'm declaring the relationship. In this case Playlists has songs and I'm basically saying that songs is a bunch of `song` (declared in `/models/Song.js`) and since it's a many-to-many relationship we are just defining how everything will look in our tables. 

So back to your question ... `via` means how we are linking them, if you go to `/models/Song.js` you will see the opposite `via: 'songs'`. I don't know if I explained myself :dancer: